### PR TITLE
Adding default value for Integer for importing

### PR DESCRIPTION
**Pull request by xbiansf**

### CODE REVIEW

File: src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java [hunk 1]
```diff
@@ -313,7 +313,7 @@ synchronized static public void registerConverters(Config cfg) {
         // Register DynaBean type conversions
         ConvertUtils.register(new DateConverter(tz, useEuroDates), Calendar.class);
         ConvertUtils.register(new DoubleConverter(), Double.class);
-        ConvertUtils.register(new IntegerConverter(), Integer.class);
+        ConvertUtils.register(new IntegerConverter(null), Integer.class);
```
Comment (xbiansf) [inline src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java]: This is used to process "#N/A" when mapped integer string to an Integer Data type in salesforce.  Given this change,  the IntegerConverter will have a default value,  and use null when it find #N/A in IntergerConverter.java#convert() method.

Comment (jjangsfdc) [inline src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java]: Do we want to set default as null or 0? Maybe I am not fully understanding the problem here...

Comment (xbiansf) [inline src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java]: null means no value provided.  0 means 0. So null should be the correct value.

Comment (jjangsfdc) [inline src/main/java/com/salesforce/dataloader/dyna/SforceDynaBean.java]: I see, so this is the case when nothing is set.

**Pull request merged**